tests/integration/test_ssm.py

```diff
@@ -144,3 +144,10 @@ def test_get_parameters_by_path_and_filter_by_labels(self, ssm_client, create_pa
         assert found_param["ARN"]
         assert found_param["Type"] == "String"
         assert found_param["Value"] == "value"
+
+    def test_get_inexistent_maintenance_window(self, ssm_client):
```

Comment:
If this was tested against AWS please mark it with `@pytest.mark.aws_validated`. The other tests in the class have the marker as far as I can see.

Feel free to merge after addressing this.

Reply:
Added.